static/src/javascripts/projects/common/modules/article/rich-links.js

```diff
@@ -135,7 +136,7 @@ const insertTagRichLink = (): Promise<void> => {
                 getSpacefinderRules(),
                 (paras: Element[]) => {
                     const html = richLinkTag({
-                        href: config.page.richLink,
+                        href: config.get('page.richLink'),
```

Comment:
What do you think about this at the top of the function:

```js
const {shouldHideAdverts, showRelatedContent, richLink, pageId} = config.get('page', {});
```

Seems to be much more DRY to me ..
```